qt/scientific_interfaces/Muon/ALCDataLoadingPresenter.cpp

```diff
@@ -348,19 +348,15 @@ void ALCDataLoadingPresenter::updateAvailableInfo() {
   m_numDetectors = ws->getInstrument()->getNumberDetectors();
 }
 
-std::string ALCDataLoadingPresenter::getPathFromFiles() {
-  std::string returnPath;
-  auto files = m_view->getFiles();
-  for (auto path : files) {
-    auto currentPath = path.substr(0, path.find_last_of("/\\"));
-    if (returnPath.empty())
-      returnPath = currentPath;
-    else if (currentPath != returnPath) {
-      returnPath = "Multiple Directories";
-      break; // End loop no need to check anymore
-    }
-  }
-  return returnPath;
+std::string ALCDataLoadingPresenter::getPathFromFiles() const {
+  const auto files = m_view->getFiles();
+  const auto first_directory = files[0u].substr(0u, files[0u].find_last_of("/\\"));
```

Comment:
Thanks for making those changes. Could we use camel case for this variable and then I'll approve this

Reply:
Oh whoops, been doing too much python recently